tests/10-vcdm2.js

```diff
@@ -284,7 +284,7 @@ describe('Verifiable Credentials Data Model v2.0', function() {
           issue(require('./input/credential-evidence-missing-type-fail.json')));
       });
       it.skip('All credentials, presentations, and encapsulated objects ' +
-        'MUST specify, or be associated with, additional more narrow types ' +
+        'SHOULD specify, or be associated with, additional more narrow types ' +
         '(like UniversityDegreeCredential, for example) so software systems ' +
         ' can process this additional information.', async function() {
```

Comment:
```suggestion
        '(like ExampleDegreeCredential, for example) so software systems ' +
        ' can more easily detect and process this additional information.', async function() {
```

May as well match the current text exactly. Helps keep things connected.